src/components/ButtonToolbar/index.jsx

```diff
@@ -5,7 +5,7 @@ import classNames from 'classnames';
 const ButtonToolbar = (props) => {
   const { className } = props;
   const fullClassName = classNames(
-    'btn-toolbar justify-content-end',
+    'btn-toolbar',
```

Comment:
Dropping that might change all the other toolbars on the site. We use it on the course edit page and the course run collapsibles, I think.

Reply:
Oh good catch, I should have seen if this had cascading changes, I'll look for a better way to override